tutorial/README.md

```diff
@@ -0,0 +1,476 @@
+
+As we see, the number of alternatives alone should not be considered as a metric to validate the quality of service of a network, as it depends on the best alternative, which, in the case of a metro line available, is much better than any other alternative!
+
+## Lesson 6: Edit existing lines
+
+In this lesson, we will learn to modify existing lines, update their schedules and validate the results again. We will extend a metro line and modify a bus line along this metro line. 
+
+### Edit existing line, to add a few stops
+
+To see all current lines, go to the `Agencies and Lines` panel, from the left menu and expand the lines by clicking the `Lines` text under the agency name.
+
+![Lesson 6: Agencies and lines panel](images/lesson6_agenciesAndLines.png)
+
+Select the metro line that will be edited (here, the Montreal STM's blue line). It will open the line edit panel, with the inbound and outbound paths for this line.
+
+![Lesson 6: Line edit panel](images/lesson6_lineEditPanel.png)
+
+Click on one of the paths to edit. They will both be edited, so any will do for now. The entire path is selected and a bottom panel opens to show the stops of the path. At the top left of the bottom panel is a help button ![path edit help button](images/buttonPathEditHelp.png) that shows information on how to edit a path. Click on it to display the help functions.
+
+![path edit bottom panel](images/lesson6_pathEditBottomPanel.png)
+
+We will extend the lines towards the northeast. Stop nodes will need to be added either at the beginning or at the end of the line, depending on its direction. 
+
+To add nodes at the end, simply click on the nodes, in order. To add the nodes at the beginning, shift-click on the node to add.
+
+Notice that since metro lines are underground, they do not follow the road network and the line will between 2 nodes will be a straight line.
+
+![Lesson 6: straight lines between metro stations](images/lesson6_straightLinesBetweenNodes.png)
+
+Now click on the save button ![save button](images/buttonSave.png) to save this path.
+
+Do the same operation for the reverse path, adding nodes at the end or beginning of the path and saving it. The paths list in the line should now show more stops on both paths.
+
+![Lesson 6: modified path list](images/lesson6_modifiedPathList.png)
+
+### Update schedules for the edited line
+
+Now that paths have been changed, the schedules should also be updated. To do so, in the `Line edit` panel, click on the `Timetables` button to open the timetable panel on the left.
+
+![Lesson 6: open timetables](images/lesson6_openTimeTables.png)
+
+Select the current service to open the current schedule. This will display all periods for the day. The current schedule comes from the imported GTFS file, so each trip may be at different times. In transit planning, fine-tuning each trip is not desirable, so we will re-create the schedule for each period with a certain interval.
```

Comment:
In transit planning, fine-tuning each trip is not desirable  -> In transit planning, fine-tuning each trip is usually not required